### PR TITLE
issue-3751 - Add missing docs, use class types, wrap load into try catch

### PR DESCRIPTION
**Pull request by AleksandrsKondratjevs**

### CODE REVIEW

File: Model/Resolver/Shipments.php [hunk 1]
```diff
@@ -41,7 +41,7 @@ public function resolve(Field $field, $context, ResolveInfo $info, array $value
         foreach ($shipments as $shipment) {
             $orderShipments[] =
                 [
-                    'id' => base64_encode($shipment->getEntityId()),
+                    'id' => base64_encode((string) $shipment->getEntityId()),
```
Comment (AleksandrsKondratjevs) [inline Model/Resolver/Shipments.php:44]: getEntityId() Int||null
base64_encode(string $string): string

File: Model/Resolver/Shipments.php [hunk 2]
```diff
@@ -24,11 +24,11 @@ class Shipments extends BaseShipments
      */
     public function resolve(Field $field, $context, ResolveInfo $info, array $value = null, array $args = null)
     {
-        if (!isset($value['model']) && !($value['model'] instanceof Order)) {
+        if (!isset($value['model']) && !($value['model'] instanceof OrderInterface)) {
```
Comment (AleksandrsKondratjevs) [inline Model/Resolver/Shipments.php:27]: Confused. In core magento modules it is checking for invoices instanceof OrderInterface, for shipments it was checking just instanceof Order :D

Comment (aleksandrsm) [reply Model/Resolver/Shipments.php:27]: I'd say give it a try and see if it breaks
Interface is safer for 3rd party extensions

File: Model/Resolver/Shipments.php (hunk 1 above)
Comment (aleksandrsm) [reply Model/Resolver/Shipments.php:44]: no whitespace after (string) would be standard, too

**Review by aleksandrsm — APPROVED** [review]

**Pull request merged**